### PR TITLE
Task/30 players list

### LINKED ISSUE #30: Create players list
- Create `PlayerItem` with:
  - image src
  - player name
  - if they are the game host
  - if they are the current player
  - if it's a placeholder

### PR DESCRIPTION
**Pull request by bellangerq**
- Create the `PlayerItem` component.
- Create the `PlayersList` container.
- Harmonize spacing in the `Lobby` page.
- Declutter the `Lobby` page so it matches with the design.

**Note 1**: finally, I like the naming `PlayerItem`. An item is a member of a list which will always be the case in the app (imho). When you see it, you know it's part of a list.

**Note 2**: there is no proper label to announce the game host as we don't know yet what will the badge made of (image, div, hidden text...).

Closes #30.

### CODE REVIEW

File: client/src/containers/PlayersList/PlayersList.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    margin-bottom: $spacing-small;
+  }
+
+  &__Grid {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__GridItem {
+    width: 45%;
```
Comment (hissalht) [inline client/src/containers/PlayersList/PlayersList.scss]: no point using `flex-wrap` if the child element has relative width as the children will never wrap

Comment (hissalht) [reply client/src/containers/PlayersList/PlayersList.scss]: An alternative could be to use the `flex` property instead in combination with a reasonably set `min-width`.

```scss
&__GridItem {
  flex: 1 0 50%;
  min-width: 15rem;
}
```

Comment (bellangerq) [reply client/src/containers/PlayersList/PlayersList.scss]: Okay I found a proper way to make it flexible so we don't need media queries. But we do need the `flex-wrap` and a `min-width`. We have 3 solutions to set the width:
- Create a new `$spacing-x` variable.
- Multiple the `$spacing-regular` variable.
- Don't use a variable at all: `15rem`.

I prefer the 2nd one except if we know we will use the new variable elsewhere.

File: client/src/components/PlayerItem/PlayerItem.test.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import PlayerItem from './PlayerItem'
+
+describe('PlayerItem', () => {
+  it('renders without crashing', () => {
+    const { container } = render(<PlayerItem name='Martin' />)
```
Comment (ThunderDev1) [inline client/src/components/PlayerItem/PlayerItem.test.tsx]: Could you choose a less annoying name?

File: client/src/containers/PlayersList/PlayersList.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <div className={classes}>
+      <h2>Joueurs</h2>
+      <ul className='PlayersList__Grid'>
+        {players.map(player => (
+          <PlayerItem
+            key={player.id}
+            className='PlayersList__GridItem'
+            name={player.name}
+            isSelf={currentPlayerId === player.id || false}
```
Comment (ThunderDev1) [inline client/src/containers/PlayersList/PlayersList.tsx]: why the default value?

File: client/src/containers/Settings/Settings.tsx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import React, { Component, ChangeEvent } from 'react'
+import classnames from 'classnames'
```
Comment (ThunderDev1) [inline client/src/containers/Settings/Settings.tsx]: Do we need a lib to concat 2 values ? Seems like overkill although the syntax is more elegant.

File: client/src/components/PlayerItem/PlayerItem.test.tsx (hunk 2 above)
Comment (bellangerq) [reply client/src/components/PlayerItem/PlayerItem.test.tsx]: 😲

File: client/src/containers/PlayersList/PlayersList.tsx (hunk 3 above)
Comment (bellangerq) [reply client/src/containers/PlayersList/PlayersList.tsx]: Mmh idk ✅

File: client/src/containers/Settings/Settings.tsx (hunk 4 above)
Comment (bellangerq) [reply client/src/containers/Settings/Settings.tsx]: Yep.

**Pull request merged**